### PR TITLE
Feature: Created view my flights

### PR DESCRIPTION
**Pull request by 96vladi**
# Description 
- My-flights view was implemented
- The following components were created with their own styles
    - BlockFlights: That stores all the data of each flight
   - FloatingB: It is the button to add more flights
   - Colors: Contains the colors that we are going to use in the development
- A defined list was created at the moment only for tests
- It was installed and made use of
react-native-vector-icons
- It does not have any functionality at the moment it is only a design
  
  ## Screenshots 
 
image

### CODE REVIEW

File: src/components/floatingB/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    return(
+        <ContainerFloating>
+            <FontAwesome 
+                name='plus' 
+                size={40} 
+                color='#fff' 
+            />
+        </ContainerFloating>
+    );
+};
```
Comment (LeyaimMagma) [inline src/components/floatingB/index.js:14]: and blank space

File: src/components/floatingB/styledComponent.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    height: 10%;
+    width: 20%;
+    position: absolute;
+    align-items: center;
+    justify-content: center;
+    border-radius: 40px;
+    top: 88%;
+    left: 40%;
+    background-color: ${Colors.blue_c};
+`;
```
Comment (LeyaimMagma) [inline src/components/floatingB/styledComponent.js:14]: add blank space

File: src/components/generals.js [hunk 3]
```diff
@@ -44,3 +44,8 @@ export const CustomInputStyle = styled.TextInput`
   padding: 10px;
   border: 1px solid;
 `;
+export const MainContainer = styled.SafeAreaView`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+`;
```
Comment (LeyaimMagma) [inline src/components/generals.js:51]: add blank space

File: src/screen/myFlights/index.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+                        />
+                    )}
+                />
+            </ContainerFlights>
+            <FloatingB />
+        </MainContainer>
+    );
+};
+
+export default MyFlights;
```
Comment (LeyaimMagma) [inline src/screen/myFlights/index.js:27]: add blank space

File: src/screen/myFlights/styledComponent.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    margin-left: 6%;
+    margin-right: 6%;
+`;
+
+export const TitleFlights = styled.Text`
+    font-size: 40px;
+    font-weight: 700;
+    margin-vertical: 1%;
+    color: ${Colors.blue_c};
+`;
```
Comment (LeyaimMagma) [inline src/screen/myFlights/styledComponent.js:16]: add blank space

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]
you forgot to add blanks in some files

**Review by Shadic78 — DISMISSED** [review]

**Review by DemisRincon — APPROVED** [review]

**Review by Shadic78 — APPROVED** [review]

**Review by EusebioAjas — APPROVED** [review]

**Review by jkzasori — APPROVED** [review]

**Review by LeyaimMagma — APPROVED** [review]

**Pull request merged**